modules/cbr-zone-module/variables.tf

```diff
@@ -216,3 +216,19 @@ variable "excluded_addresses" {
     error_message = "Value should be a valid as per the type"
   }
 }
+
+variable "zone_id" {
+  type = string
+  validation {
+    condition     = var.zone_id == null || (can(regex("^[0-9a-fA-F]{32}$", var.zone_id)))
+    error_message = "Value should be a valid zone ID with 32 alphanumeric characters"
+  }
+  description = "Provide an existing CBR zone ID"
+  default     = null
+}
+
+variable "use_existing_cbr_zone" {
+  type        = bool
+  description = "Whether to update CBR zone using existing zone ID"
```

Comment:
We are calling it as update existing zone - is it only append or allows modification in the existing one too?
I understand that it is only append....so need to make it clear either in readme or here in variable desc. But this info should be IN.